Datasources: Add toggle to control default behaviour of 'Manage alerts via Alerts UI' toggle (#98441)
grafana/grafana: docs/sources/datasources/prometheus/configure-prometheus-data-source.md

- **Manage alerts via Alerting UI** - Toggle to enable `Alertmanager` integration for this data source.

{{% admonition type="note" %}}

> The `Manage alerts via Alerting UI` toggle will be enabled by default. You can change this behavior by setting the [default_manage_alerts_ui_toggle]({{< relref "../../setup-grafana/configure-grafana/#default_manage_alerts_ui_toggle" >}}) option in the Grafana configuration file.

Reviewer: ```suggestion
The **Manage alerts via Alerting UI** toggle is enabled by default. You can change this behavior by setting the [default_manage_alerts_ui_toggle]({{< relref "../../setup-grafana/configure-grafana/#default_manage_alerts_ui_toggle" >}}) option in the Grafana configuration file.
```

{{% /admonition %}}

### Interval behavior

> - **Scrape interval** - Set this to the typical scrape and evaluation interval configured in Prometheus. The default is `15s`.

Reviewer: ```suggestion
- **Scrape interval** - Set to the typical scrape and evaluation interval configured in Prometheus. The default is `15s`.
```
